lib/client.js

```diff
@@ -12,42 +12,35 @@ const telemetryHeader = {
   }
 };
 
-custom.setHttpOptionsDefaults({
-  headers: {
-    'User-Agent': `${pkg.name}/${pkg.version}`,
-    'Auth0-Client': Buffer.from(JSON.stringify(telemetryHeader)).toString('base64')
-  },
-  timeout: 4000
-});
-
 async function get(config) {
-  const authorizeParams = config.authorizationParams;
 
   const issuer = await Issuer.discover(config.issuerBaseURL);
 
-  if (Array.isArray(issuer.id_token_signing_alg_values_supported) &&
-    !issuer.id_token_signing_alg_values_supported.includes(config.idTokenAlg)) {
-    throw new Error(`The issuer doesn't support the ID token algorithm ${config.idTokenAlg}
-Supported types:
-- ${issuer.id_token_signing_alg_values_supported.sort().join('\n- ')}
-`);
+  const issuerTokenAlgs = Array.isArray(issuer.id_token_signing_alg_values_supported) ? 
```

Comment:
No functionality changes, just more clear logic here and improved error message when thrown.